force-app/main/default/classes/AdvancedSearchController.cls

```diff
@@ -0,0 +1,25 @@
+public with sharing class AdvancedSearchController {
```

Comment:
The list data structure is a class and you want to capitalize the first letter of classes that you use... so you should update list to be List instead. The same is true for String. Unlike in Java there aren't primitives in apex so most of those types you've used in the past need to be capitalized when you use them (String, Integer, etc..)